.travis.yml

```diff
@@ -0,0 +1,29 @@
+# Copyright (c) 2021 Oracle and/or its affiliates. All rights reserved.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Eclipse Distribution License v. 1.0, which is available at
+# http://www.eclipse.org/org/documents/edl-v10.php.
+#
+# SPDX-License-Identifier: BSD-3-Clause
+#
+
+dist: focal
+
+sudo: required
+
+language: java
+
+services:
+  - docker
```

Comment:
What is it used for?

Reply:
java is needed to have some tools installed, like maven, JDK, etc.

Docker was a copy-paste. It seems it is not required so I have removed it.